src/js/components/search/FilterExpandButton.jsx

```diff
@@ -25,10 +25,32 @@ export default class FilterExpandButton extends React.Component {
             icon = <Icons.AngleDown />;
         }
 
+        let clickEvent = '';
+        switch (this.props.name) {
+            case 'Search':
+                clickEvent = null;
+                break;
+            case 'Budget Categories':
+                clickEvent = null;
+                break;
+            case 'Award ID':
+                clickEvent = null;
+                break;
+            case 'Award Amount':
+                clickEvent = null;
+                break;
+            case 'Other Award Items':
+                clickEvent = null;
+                break;
+            default:
+                clickEvent = this.props.toggleFilter;
+                break;
```

Comment:
Since clickEvent is null in all but the default case, you might consider letting your initial declaration be null, and only declare it again in the default case.  If it's initially set to null, you won't need to re-set it unless it is something other than null.